### PR TITLE
[VI] Updated: Broadcasts 2 Vietnamese translation

### PR DESCRIPTION
**Pull request by rungxanh2901**
My attempt to help localizing Broadcasts to Vietnamese-speaking users.
Update on 2020-11-14: added extra strings for Broadcasts 2

### CODE REVIEW

**Comment by steventroughtonsmith** [comment]
Thanks for these! Attached are some screenshots of how it looks right now — please let me know if anything needs to be changed, and if it's all good then I'm happy to merge

Screenshot 2020-11-18 at 00 41 04
Screenshot 2020-11-18 at 00 40 50
Screenshot 2020-11-18 at 00 40 33
Screenshot 2020-11-18 at 00 39 20
Screenshot 2020-11-18 at 00 39 14
Screenshot 2020-11-18 at 00 39 05
Screenshot 2020-11-18 at 00 38 53

**Comment by rungxanh2901** [comment]
> Thanks for these! Attached are some screenshots of how it looks right now — please let me know if anything needs to be changed, and if it's all good then I'm happy to merge
> 
> Screenshot 2020-11-18 at 00 41 04
> 
> Screenshot 2020-11-18 at 00 40 50
> 
> Screenshot 2020-11-18 at 00 40 33
> 
> Screenshot 2020-11-18 at 00 39 20
> 
> Screenshot 2020-11-18 at 00 39 14
> 
> Screenshot 2020-11-18 at 00 39 05
> 
> Screenshot 2020-11-18 at 00 38 53
> 
> Screenshot 2020-11-18 at 00 43 23

Dear Steven, I have just pushed my updated work after getting your screenshots. Thank you for considering my contributions, and hope to see this go live in the update! I'll keep updating for any possible improvements

File: vi.lproj/Localizable.strings [hunk 1]
```diff
@@ -194,18 +194,18 @@
...
 "SLEEP_TIMER_MENU_15" = "15 phút";
 "SLEEP_TIMER_MENU_30" = "30 phút";
 "SLEEP_TIMER_MENU_60" = "60 phút";
 
-"EDIT_STATION_WINDOW_NAME_FIELD_LABEL" = "tên";
-"EDIT_STATION_WINDOW_URL_FIELD_LABEL" = "địa chỉ";
-"EDIT_STATION_WINDOW_IMAGE_FIELD_LABEL" = "ảnh artwork";
+"EDIT_STATION_WINDOW_NAME_FIELD_LABEL" = "Tên đài";
+"EDIT_STATION_WINDOW_URL_FIELD_LABEL" = "Địa chỉ";
+"EDIT_STATION_WINDOW_IMAGE_FIELD_LABEL" = "Ảnh đại diện";
```
Comment (steventroughtonsmith) [inline vi.lproj/Localizable.strings:205]: In English, I'm leaving these without capitalization, which is how the Music app shows its metadata/info panel in macOS 11. I'm happy either way, though!

**Comment by rungxanh2901** [comment]
I've just changed the capitalization of these 3 lines to suit your intended wordings. Amazing attention to detail man 👌🏻

**Comment by steventroughtonsmith** [comment]
Sweet! Thanks 😄 I'll merge this now and I'll give it a full run through and make sure everything resizes properly to fit. Will be included in Broadcasts v2.0 (macOS 11 first, iOS 14 later).

**Pull request merged**